<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CEPLs-MacBook-Air.local" failures="0" tests="2" name="com_powderRoom_LoginTest.UserLoginTest" time="22.971" errors="1" timestamp="2025-08-15T22:46:59 IST" skipped="1">
  <testcase classname="com_powderRoom_LoginTest.UserLoginTest" name="TC_Wl_01_welcomeScreenTest" time="0.000">
    <skipped/>
  </testcase> <!-- TC_Wl_01_welcomeScreenTest -->
  <testcase classname="com_powderRoom_LoginTest.UserLoginTest" name="setup" time="22.971">
    <error message="Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20000ms 
Host info: host: &#039;CEPLs-MacBook-Air.local&#039;, ip: &#039;fe80:0:0:0:c7a:d309:7f0:57a6%en0&#039;
Build info: version: &#039;4.13.0&#039;, revision: &#039;ba948ece5b*&#039;
System info: os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;15.3.2&#039;, java.version: &#039;21.0.6&#039;
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{appium:udid=07589311BN040005, appium:autoGrantPermissions=true, appium:deviceName=INFINIX MOBILITY LIMITED infinix X663 API 31, platformName=ANDROID, appium:automationName=UiAutomator2, appium:app=/Users/cepl/git/WolooPowderRoom1/PowderRoom_Automation/apps/app-woloo-debug (5).apk, appium:platformVersion=12}], desiredCapabilities=Capabilities {appium:app: /Users/cepl/git/WolooPowder..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: INFINIX MOBILITY LIMITED in..., appium:platformVersion: 12, appium:udid: 07589311BN040005, platformName: ANDROID}}]
Capabilities {appium:app: /Users/cepl/git/WolooPowder..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: INFINIX MOBILITY LIMITED in..., appium:platformVersion: 12, appium:udid: 07589311BN040005, platformName: ANDROID}" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20000ms 
Host info: host: 'CEPLs-MacBook-Air.local', ip: 'fe80:0:0:0:c7a:d309:7f0:57a6%en0'
Build info: version: '4.13.0', revision: 'ba948ece5b*'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '15.3.2', java.version: '21.0.6'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{appium:udid=07589311BN040005, appium:autoGrantPermissions=true, appium:deviceName=INFINIX MOBILITY LIMITED infinix X663 API 31, platformName=ANDROID, appium:automationName=UiAutomator2, appium:app=/Users/cepl/git/WolooPowderRoom1/PowderRoom_Automation/apps/app-woloo-debug (5).apk, appium:platformVersion=12}], desiredCapabilities=Capabilities {appium:app: /Users/cepl/git/WolooPowder..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: INFINIX MOBILITY LIMITED in..., appium:platformVersion: 12, appium:udid: 07589311BN040005, platformName: ANDROID}}]
Capabilities {appium:app: /Users/cepl/git/WolooPowder..., appium:autoGrantPermissions: true, appium:automationName: UiAutomator2, appium:deviceName: INFINIX MOBILITY LIMITED in..., appium:platformVersion: 12, appium:udid: 07589311BN040005, platformName: ANDROID}
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:140)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:133)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:102)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:182)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:274)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:163)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:89)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:101)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:113)
at Utilities.BaseClass.setup(BaseClass.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setup -->
</testsuite> <!-- com_powderRoom_LoginTest.UserLoginTest -->
